frontend/src/metabase/query_builder/actions/object-detail.js

```diff
@@ -56,13 +57,31 @@ export const followForeignKey = createThunkAction(
         return false;
       }
 
-      const newCard = startNewCard("query", card.dataset_query.database);
-
-      newCard.dataset_query.query["source-table"] = fk.origin.table.id;
-      newCard.dataset_query.query.filter = getFilterForFK(objectId, fk);
+      const databaseId = card.dataset_query.database;
+      const tableId = fk.origin.table.id;
+      const metadata = getMetadata(getState());
+      const question = Question.create({ databaseId, tableId, metadata });
```

Comment:
could you please clarify why `Question.create` is used instead of new Question?

Reply:
9c6a7ecc183337644a075ba17b57e94874905ea5

Updated to not pass `tableId` to `Question.create`. For context see [Slack discussion](https://metaboat.slack.com/archives/C0645JP1W81/p1705322543442019).